### DISCUSSION TITLE
Problem with verifing

### DISCUSSION THREAD

**Discussion by mirotec** [community/community#147814] [GitHub Education]
### Hi everyone,

So, my problem is when i make a submission for student on github.
I got response that my last name is wrong in billing details.
But my last name is same as it is in paper from school.
Paper is readable and can´t contact github edu support directly.

Thanks

**Comment by OtayNacef** [comment]
Double-check your billing details to make sure they match exactly with what’s on your school documents,
even tiny differences like capitalization or extra spaces can sometimes cause issues.
You can update or review your billing info here:
[Check your account billing](https://github.com/settings/billing/payment_information)
